semantics/language/common/expr/eval.k

```diff
@@ -172,14 +172,14 @@ module C-COMMON-EXPR-EVAL
 
      syntax Error ::= "errorLocInvalid"
                     | "errorLocBounds"
-     rule (.K => UNDEF("CEE2",
+     rule (.K => UNDEF("CEE4",
                "Referring to an object outside of its lifetime.",
-               "6.2.4"))
+               "6.2.4, J.2:1 item 9"))
           ~> errorLocInvalid
           [structural]
      rule (.K => UNDEF("CEE3",
                "Found pointer that refers outside the bounds of an object + 1.",
-               "6.2.4"))
+               "6.7.2.1, J.2:1 item 62"))
```

Comment:
This is not correct; item 62 deals with flexible array members, which this is not.  I think you want item 19, which is paragraph 6.3.2.1:1